paasta_tools/marathon_tools.py

```diff
@@ -298,7 +298,7 @@ def get_drain_method_params(self, service_namespace_config):
             default = {'delay': 30}
         return self.config_dict.get('drain_method_params', default)
 
-    def get_constraints(self, service_namespace_config):
+    def get_calculated_constraints(self, service_namespace_config):
         """Gets the constraints specified in the service's marathon configuration.
 
         These are Marathon app constraints. See
```

Comment:
please update this docstring - the notes about defaults now belong with ``get_constraints`` - this docstring should describe the different sets of constraints it merges